docker-compose.yml

```diff
@@ -20,6 +20,7 @@ keycloak:
   volumes:
   - ./.docker-compose/keycloak.env:/etc/secrets/keycloak.env
   - ./govuk/:/opt/jboss/keycloak/themes/govuk/
+  - ./govuk-social-provider-login/:/opt/jboss/keycloak/themes/govuk-social-provider-login/
```

Comment:
We’re just lucky we don‘t have any caches to invalidate or any risk of off-by-two errors.